pycobertura/cobertura.py

```diff
@@ -77,32 +77,50 @@ def _load_from_file(self, report_file):
     def _load_from_string(self, s):
         return ET.fromstring(s)
 
+    @memoize
+    def _get_class_elements_by_filename(self, filename):
+        syntax = f"./packages//class[@filename='{filename}']"
+        classes = []
+        for c in self.xml.xpath(syntax):
+            classes.append(c)
```

Comment:
Can we change this code to be consistent with the variable names used in `_get_lines_by_filename()`?

```suggestion
        classElements = []
        for classElement in self.xml.xpath(syntax):
            classElements.append(classElement)
```

Alternatively, if `classElement` is too verbose, I'd be okay to use `classEl`.

Reply:
done in https://github.com/aconrad/pycobertura/pull/132/commits/45e1b7b103bd59c9ff4756375a69fbcd479d0608